modules/rts/rts.commander.ts

```diff
@@ -41,9 +41,11 @@ export class RedisTimeSeriesCommander {
      * @param key Required. The key
      * @param retention Optional. The retention time
      * @param labels Optional. The labels to update
+     * @param duplicatePolicy Optional. An update to the duplicate policy
+     * @param chunkSize Optional. An update to the chunk size
      * 
      */
-    alter(key: string, retention?: number, labels?: TSLabel[]): CommandData {
+    alter(key: string, retention?: number, labels?: TSLabel[], duplicatePolicy?: string, chunkSize?: number): CommandData {
```

Comment:
Can you please move all the optional parameters into a type (you can look across the project for more examples)? and then instead of these parameters please add 1 parameters called options as `options?: YourNewType`.
If you need more guidance, I can help